### PR TITLE
(DOCSP-28387)(DOCSP-28289) Adds troubleshooting page

### PR DESCRIPTION
**Pull request by sarahsimpers**
[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64060c80054271fb0aa1717d)
[Jira Ticket 1](https://jira.mongodb.org/browse/DOCSP-28387)
[Jira Ticket 2](https://jira.mongodb.org/browse/DOCSP-28289)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-28387/troubleshooting/)

### CODE REVIEW

**Review by jwilliams-mongo — APPROVED** [review]
@sarahsimpers two style nits. LGTM otherwise

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: source/troubleshooting.txt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- Run ``atlas auth login`` to authenticate using your |service| login
+  credentials and an authentication token.
+
+To learn more, see :ref:`connect-atlas-cli`.
+
+atlas: command not found
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``atlas`` executable might be in a directory which is not in your
+path. You can either add the directory to your path, move the executable
```
Comment (gssbzn) [inline source/troubleshooting.txt:39]: should this be 
```suggestion
$PATH. You can either add the directory to your path, move the executable
```
to make it clear what path?

Comment (sarahsimpers) [reply source/troubleshooting.txt:39]: Sure, will change it. Thanks!

**Pull request merged**